src/biogeophys/SoilHydrologyMod.F90

```diff
@@ -219,165 +274,71 @@ subroutine Infiltration(bounds, num_hydrologyc, filter_hydrologyc, num_urbanc, f
-     real(r8) :: alpha_evap(bounds%begc:bounds%endc)        ! fraction of total evap from h2osfc
-     real(r8) :: qflx_evap(bounds%begc:bounds%endc)         ! local evaporation array
      real(r8) :: qflx_h2osfc_drain(bounds%begc:bounds%endc) ! bottom drainage from h2osfc
-     real(r8) :: qflx_in_h2osfc(bounds%begc:bounds%endc)    ! surface input to h2osfc
-     real(r8) :: qflx_in_soil(bounds%begc:bounds%endc)      ! surface input to soil
-     real(r8) :: qflx_infl_excess(bounds%begc:bounds%endc)  ! infiltration excess runoff -> h2osfc
+     real(r8) :: qflx_in_h2osfc(bounds%begc:bounds%endc)    ! net surface input to h2osfc
      real(r8) :: frac_infclust                              ! fraction of submerged area that is connected
-     real(r8) :: fsno                                       ! copy of frac_sno
      real(r8) :: k_wet                                      ! linear reservoir coefficient for h2osfc
-     real(r8) :: fac                                        ! soil wetness of surface layer
-     real(r8) :: psit                                       ! negative potential of soil
-     real(r8) :: hr                                         ! relative humidity
-     real(r8) :: wx                                         ! partial volume of ice and water of surface layer
-     real(r8) :: z_avg
-     real(r8) :: rho_avg
-     real(r8) :: fmelt
-     real(r8) :: f_sno
-     real(r8) :: imped
-     real(r8) :: d
-     real(r8) :: h2osoi_vol                                 
-     real(r8) :: basis                                      ! temporary, variable soil moisture holding capacity 
-     ! in top VIC layers for runoff calculation
-     real(r8) :: rsurf_vic                                  ! temp VIC surface runoff
-     real(r8) :: i_0(bounds%begc:bounds%endc)               ! average soil moisture in top VIC layers (mm)
-     real(r8) :: max_infil(bounds%begc:bounds%endc)         ! max infiltration capacity in VIC (mm)
-     real(r8) :: top_icefrac                                ! temporary, ice fraction in top VIC layers
      !-----------------------------------------------------------------------
 
      associate(                                                        & 
           snl              =>    col%snl                             , & ! Input:  [integer  (:)   ]  minus number of snow layers                        
-          dz               =>    col%dz                              , & ! Input:  [real(r8) (:,:) ]  layer depth (m)                                 
 
-          t_soisno         =>    temperature_inst%t_soisno_col       , & ! Input:  [real(r8) (:,:) ]  soil temperature (Kelvin)                       
+          qflx_infl_excess =>    infiltration_excess_runoff_inst%qflx_infl_excess_col , & ! Input:  [real(r8) (:)] infiltration excess runoff (mm H2O /s)
+          qinmax           =>    infiltration_excess_runoff_inst%qinmax_col , & ! Input:  [real(r8) (:)] maximum infiltration capacity (mm H2O /s)
```

Comment:
**Comment from @martynpclark Aug 24, 2017**

Comment should be the maximum infiltration rate